src/main/java/mate/academy/util/UtilConnection.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class UtilConnection {
```

Comment:
it's usually like this, first is the name of an entity, then what it does, or what is it's point, like `TransactionExecutor`, `DataParser`, `ClientProvider`, `ValidationUtils` or `ValidationUtil` etc. 
```suggestion
public class ConnectionUtil {
```